swagger.yaml

```diff
@@ -242,6 +242,26 @@ paths:
           schema:
             type: string
             format: uuid
+        - name: firstResult
+          in: query
+          description: index of the first result
+          schema:
+            type: integer
+        - name: maxResults
+          in: query
+          description: limit results to this amount
+          schema:
+            type: integer
+        - name: sortByDateOldestFirst
+          in: query
+          description: Sort by date, return the oldest entry first
+          schema:
+            type: boolean
+        - name: sortByDateNewestFirst
+          in: query
+          description: Sort by date, return the newest entry first
+          schema:
+            type: boolean
```

Comment:
As a default we should always return the newest items first so no need for this.
```suggestion
```